README.md

```diff
@@ -42,6 +43,14 @@ const server = new ApolloServer({
 export default startServerAndCreateLambdaHandler(
   server,
   handlers.createAPIGatewayProxyEventV2RequestHandler(),
+  {
+    context: async ({ event }) => {
+      return {
+        myCustomAuthContext: event.requestContext.authorizer,
+      };
+    },
+    middleware: [customMiddleware],
+  },
```

Comment:
Could we move this example down to it's own block? This first example is meant to be a basic getting started block.